release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2515,6 +2515,8 @@ openshift.io/scc: hostmount-anyuid
 
 * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
 
+* When deploying a Bare Metal machine on {rh-openstack}, customers must set the value of `force_metadata` to `False` in the Neutron DHCP agent configuration to overcome a link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[known issue].
```

Comment:
@JoeAldinger Assuming this makes it through change control, is this the appropriate place to add this item?

Reply:
/lgtm